src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -165,5 +165,6 @@ static constexpr ov::Property<size_t> rng_seed{"rng_seed"};
 OPENVINO_GENAI_EXPORTS GenerationConfig beam_search();
 OPENVINO_GENAI_EXPORTS GenerationConfig greedy();
 OPENVINO_GENAI_EXPORTS GenerationConfig multinomial();
+OPENVINO_GENAI_EXPORTS GenerationConfig vlm_multinomial();
```

Comment:
let's avoid extending pre-configured generation configs set

actually, I want to deprecate old ones to encourage users to specify parameters explicitly